R/mwrite.R

```diff
@@ -9,7 +9,18 @@ tocode <- function(l) {
 
 get_upper_tri <- function(x) {
```

Comment:
This now returns a list with the rows of each matrix in lower-triangular format: 

```r
list(
1, 
c(2,3), 
c(4,5,6)
)
```
etc.